### DISCUSSION TITLE
Manual setup of sales channels

### DISCUSSION THREAD

**Discussion by Brunounyime** [medusajs/medusa#7043] [Q&A]
Please how do i manually set up or create my sales channel? I need multiple sales channels, I can create a new sales channel from the admin but I can't find it and I don't really understand how it works.

**Comment by terribilis** [comment]
Within the Admin site you can navigate to the settings. There navigate to Sales Channel, select the '+' icon and you will be able to make a new sales channel.

**Comment by Brunounyime** [comment]
Thanks Medusajs/Medusa

What I intend to achieve is after  I create the new sales channel I want to
be able to view the products of the new sales channel likewise the default
sales channel, like switching between sales channels.
